functions-continued/practice-functions-continued.md

```diff
@@ -0,0 +1,525 @@
+| `"Bulbasaur"`    | `"Pikachu"`      | `"Bulbasaur"`        |
+| `"Pikachu"`      | `"Pikachu"`      | `"Pikachu"`          |
+| `"Pikachu"`      | `"pikachu"`      | `"Pikachu"`          |
+
+### !end-question
+### !tests
+```python
+import unittest
+from main import *
+
+class TestChallenge(unittest.TestCase):
+    def test_compare_alphabetically(self):
+        self.assertEqual(compare_alphabetically("Pikachu", "Bulbasaur"), "Bulbasaur")
+        self.assertEqual(compare_alphabetically("Bulbasaur", "Bulbasaur"), "Bulbasaur")
+        self.assertEqual(compare_alphabetically("Pikachu", "Pikachu"), "Pikachu")
+        self.assertEqual(compare_alphabetically("Pikachu", "pikachu"), "Pikachu")
+```
+### !end-tests
+### !explanation
+
+An example of a working implementation:
+
+```python
+def compare_alphabetically(word_a, word_b):
+    if word_a < word_b:
+        return word_a
+    else:
+        return word_b
+```
+
+### !end-explanation
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 9 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: code-snippet
+* language: python3.6
+* id: kJYUgL
+* title: Functions, continued
+### !question
+
+Create a function named `compare_values`. It has two parameters, `a` and `b`, which will be numbers. Compare `a` against `b`. If `a` is less than `b`, return `-1`. If `a` is equal to `b`, return `0`. If `a` is greater than `b`, return `1`.
+
+| Example `a` | Example `b` | Example return value |
+| ----------- | ----------- | -------------------- |
+| `999`       | `-999`      | `1`                  |
+| `-5`        | `-5`        | `0`                  |
+| `80.5`      | `80.6`      | `-1`                 |
+
+### !end-question
+### !tests
+```python
+import unittest
+from main import *
+
+class TestChallenge(unittest.TestCase):
+    def test_compare_values(self):
+        self.assertEqual(compare_values(999, -999), 1)
+        self.assertEqual(compare_values(-5, -5), 0)
+        self.assertEqual(compare_values(80.5, 80.6), -1)
+```
+### !end-tests
+### !explanation
+
+An example of a working implementation:
+
+```python
+def compare_values(a, b):
+    if a < b:
+        return -1
+    elif a == b:
+        return 0
+    else:
+        return 1
+```
+
+### !end-explanation
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 10 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: code-snippet
+* language: python3.6
+* id: Sl6onl
+* title: Functions, continued
+### !question
+
+Create a function named `fizz_buzz`. It has one parameter, `num`.
+
+If `num` is a multiple of 3, it should return `"Fizz"`. If `num` is a multiple of 5, it should return `"Buzz"`. If `num` is a multiple of 3 _and_ 5, it should return `"FizzBuzz"`. If `num` is _not_ a multiple of either 3 or 5, it should return `num`.
+
+| Example `num` | Example return value |
+| ------------- | -------------------- |
+| `3`           | `"Fizz"`             |
```

Comment:
Add a neither 3 nor 5 sample
```suggestion
| `1`           | `1`                  |
| `3`           | `"Fizz"`             |
```

Reply:
Manually editing your suggestion to add `2` while we're at it